report_thesis/src/sections/baseline.tex

```diff
@@ -0,0 +1,52 @@
+In \citet{p9_paper}, we presented our efforts to replicate the MOC model by \citet{cleggRecalibrationMarsScience2017}.
+This effort was motivated by our desire to understand the model and its performance better, and to experiment with its components to determine how it could be improved.
+However, as discussed, there were some differences between our replica and the original model.
+These differences were caused by missing information in the original paper, and so rather than introducing our own assumptions, we designed experiments to determine the best way to replicate the model.
+
+initially, our replication omitted the use of the \gls{mad} for outlier removal in the data preprocessing stage, a step included in the original model.
+The omission was due to the lack of information regarding its implementation within the processing pipeline.
+To rectify this, we experimented with applying \gls{mad} at various stages in the pipeline.
+We determined that its application post-removal of the initial five shots, and prior to masking and normalization, most closely mirrored the outcomes of the original model.
+
+Furthermore, our replica only utilized a single dataset for the \gls{ica} phase, while the original model used all five datasets.
+This difference was due to the original paper not specifying how the five datasets were used, and so we designed an experiment to determine how to use them in a way that would most closely replicate the original model.
+We initially assumed that the datasets were aggregated and used as a single dataset.
+This approach, however, did not align with the original model's results, likely due to the loss of information from the individual datasets.
+Following this discovery, we instead used the datasets in the same way as we did in the \gls{pls1-sm} phase, which yielded results aligning more closely with the original model.
+
+Finally, our original replica used a random train/test split for training, in contrast to the original model's manual curation to ensure representation of extreme compositions in both sets.
```

Comment:
`original` followed by `replica` is a little funny

```suggestion
Finally, our initial replication used a random train/test split for training, in contrast to the original model's manual curation to ensure representation of extreme compositions in both sets.
```

Reply:
Why? Also "replica" is what we called it in the previous paper, so why change it to replication here?